pkg/printers/internalversion/printers.go

```diff
@@ -2494,7 +2496,31 @@ func printPriorityLevelConfiguration(obj *flowcontrol.PriorityLevelConfiguration
 			queueLengthLimit = qc.QueueLengthLimit
 		}
 	}
-	row.Cells = append(row.Cells, name, string(obj.Spec.Type), acs, queues, handSize, queueLengthLimit, translateTimestampSince(obj.CreationTimestamp))
+	var someUnlimited bool
+	var minLimit int32 = math.MaxInt32
+	var maxLimit int32
```

Comment:
what's a typical number of concurrency limits? if it's just a handful, does it make sense to simplify this and print the whole list? I'm not too familiar with some example configs for PL, so please ignore if it doesn't make sense to print all limits

Reply:
The multiplicity here is due to the fact that there can be multiple apiservers; each can potentially report a different concurrency limit.  It is highly likely that many apiservers will report the same concurrency limit.  If printing the whole list, I would want to remove duplicates.  I think that would make it more complicated than the code here now.